### ISSUE TITLE
"IRN: Adblock-Iran" Regional filterlist breakage

### ISSUE THREAD

**Issue by MasterKia** [uBlockOrigin/uAssets#13253]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`http://www.ashoora.ir`
`https://bayanbox.ir`
`https://asriran.com` (`http://darunameh.blogfa.com/post/16`)

### Describe the issue

[IRN: Adblock-Iran](https://github.com/farrokhi/adblock-iran) is one of the regional filterlists in uBO.

* `ashoora.ir`:
The `.ir/*/banners/` filter from _Adblock-Iran_ blocks `ashoora.ir/images/banners/Rahyan_410.gif` and `ashoora.ir/images/banners/Behesht_410.gif` at the bottom of the page.
https://user-images.githubusercontent.com/17685483/168642056-69a0aaf3-113d-44ec-9f5b-bbf8f2f2925b.png

* `bayanbox.ir` (Upload Site):
It's a regular **upload** site, where you can upload stuff for free.
The `||bayanbox.ir/view/*/*.gif` filter from _Adblock-Iran_ blocks normal gifs like `https://bayanbox.ir/view/6514224053252251384/Quby-Sticker-2-12.gif` and `https://bayanbox.ir/view/4639030023267517922/Kak-Eunsang.gif` (I forgot which page those gifs were on, but overall this filter is unnecessary).
https://user-images.githubusercontent.com/17685483/168645566-ae6e3617-cf06-4bb8-ae01-c24413ae1caa.png

* `asriran.com`:
It's a **news** site.
The `||cdn.asriran.com^$third-party` filter from _Adblock-Iran_ wrongly blocks `https://cdn.asriran.com/files/fa/news/1398/6/29/1017722_194.jpg`.
https://user-images.githubusercontent.com/17685483/168646580-6d859e0b-3690-4cee-994c-f3068c0a1b1f.png

### Screenshot(s)

`ashoora.ir`:
https://user-images.githubusercontent.com/17685483/168642139-c79f6336-365e-4390-8d46-d0b6c8f8181b.png

### Notes

These filters shouldn't exist:
```
.ir/*/banners/
||bayanbox.ir/view/*/*.gif
||cdn.asriran.com^$third-party
```

### Configuration

<details>

```yaml
uBlock Origin: 1.42.4
Chromium: 101
filterset (summary): 
  network: 83580
  cosmetic: 71753
  scriptlet: 18184
  html: 0
listset (total-discarded, last updated): 
  added: 
    fanboy-cookiemonster: 31879-83, 19m
  default: 
    user-filters: 0-0, never
    easylist: 66428-51, 2d.21h.53m
    easyprivacy: 26943-121, 3d.3h.5m
    plowe-0: 3676-805, 9d.5h.47m
    ublock-abuse: 75-0, 3d.3h.4m
    ublock-badware: 4149-92, 3d.3h.4m
    ublock-filters: 30970-98, 3d.3h.4m
    ublock-privacy: 215-2, 3d.3h.4m
    ublock-quick-fixes: 220-0, 19m
    ublock-unbreak: 1779-41, 2d.20h.18m
    urlhaus-1: 8632-0, 19m
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 235 ms (selfie)
  maxAssetCacheWait: 219 ms
```
</details>

**Comment by MasterKia** [comment]
**Side note:**
The current regional Persian filterlist ([adblock-iran](https://github.com/farrokhi/adblock-iran)) isn't maintained (last commit: 5 months ago) unlike other regional lists (For instance: Albania, Indonesian, Indian, Korean, Lithuania and Dutch all have been updated in the last week).
Also the maintainer of _adblock-iran_ has been [nonresponsive to issues](https://github.com/farrokhi/adblock-iran/issues?q=is%3Aopen+is%3Aissue) in the last 5 months (12 open issues, 7 of them submitted by me).

I propose adding the [uBOPa (uBO Parsi/Persian)](https://github.com/nimasaj/uBOPa) filterlist.
uBOPa's maintainer has been responsive to all 46 [issues I've submitted](https://github.com/nimasaj/uBOPa/issues?q=is%3Aissue+author%3AMasterKia+is%3Aclosed) and he updates the list frequently.

Otherwise since _adblock-iran_ is abandoned, I'd have to submit all persian regional issues here.

**Comment by MasterKia** [comment]
@nimasaj

**Comment by MasterKia** [comment]
> We know this list already but the list contains many problematic filters like `*##*[class*=" adv "]`

If he removes problematic (mostly cosmetic) filters, is there a chance of adding this list?

**Comment by Yuki2718** [comment]
No, not only cosmetic. We really can't accept such way too generic rules like `*-ads-^$script` `||ad0*` `*_adv-^$image`. Note `||ad0*` is not even tokenizable.

**Comment by MasterKia** [comment]
I assume he'd be willing to get rid of those filters you mentioned and only include network filters and non-generic cosmetic rules.

The Adblock-Iran has some redundant or problematic filters too
Some redundant filters on top of my head:
https://github.com/farrokhi/adblock-iran/blob/master/filter.txt#L684-L703
https://github.com/farrokhi/adblock-iran/blob/master/filter.txt#L1263
https://github.com/farrokhi/adblock-iran/blob/master/filter.txt#L280

As for problematic filters, for instance it has 183 ".gif" filters for no apparent reason.
https://user-images.githubusercontent.com/17685483/168655887-bfaea245-f818-4a59-b70c-efe318e0d8be.png

Can't a website upload anything other than ads as a gif?

**Comment by JobcenterTycoon** [comment]
.gif can have animations so mostly it get used for ads, thats why.

**Comment by MasterKia** [comment]
Fair enough.
If _Adblock-Iran_ is gonna stay unmaintained, do you suggest merging future Persian filters into **uBlock filters** or **EasyList Ads/Privacy**?
